presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -86,9 +87,12 @@ public BridgingHiveMetastore(ThriftMetastore delegate)
     public Optional<Table> getTable(HiveIdentity identity, String databaseName, String tableName)
     {
         return delegate.getTable(identity, databaseName, tableName).map(table -> {
-            if (isAvroTableWithSchemaSet(table) || isCsvTable(table)) {
```

Comment:
Also update https://prestosql.io/docs/current/connector/hive.html#metastore-configuration-for-avro-and-csv

it's no longer required for CSV.

Reply:
👍